### ISSUE TITLE
Pre-Populate Value in field settings does not work with hidden field

### ISSUE THREAD

**Issue by SanboKyo** [verbb/formie#430]
The same query parameter in pre-populate value in field settings: does not work with hidden field, but works with others (for example: SingleLineText or MultiLineText)

**Additional info**
- Plugin version: 1.3.27
- Craft version: 3.6.11.2
- Multi-site: Yes

**Comment by engram-design** [comment]
Yeah, this should actually be removed, in favour of the "Query Parameter" option for the default value.

![image](https://user-images.githubusercontent.com/1221575/114476414-813b0900-9c3d-11eb-8d48-859e42933bf1.png)

**Comment by SanboKyo** [comment]
Ah, okay!

Another small bug: When I leave the query parameter field empty and want to save the form, the spinning wheel spins and spins ... nothing else happens, no error message, no saving.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.27",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.0](https://github.com/verbb/formie/releases/tag/1.4.0)
